components/sidebar/sidebar_service.cc

```diff
@@ -684,11 +684,11 @@ SidebarItem SidebarService::GetBuiltInItemForType(
     case SidebarItem::BuiltInItemType::kChatUI: {
 #if BUILDFLAG(ENABLE_AI_CHAT)
       if (ai_chat::features::IsAIChatEnabled()) {
-        return SidebarItem::Create(
-            brave_l10n::GetLocalizedResourceUTF16String(IDS_CHAT_UI_TITLE),
-            SidebarItem::Type::kTypeBuiltIn,
-            SidebarItem::BuiltInItemType::kChatUI,
-            /* open_in_panel = */ true);
+        return SidebarItem::Create(brave_l10n::GetLocalizedResourceUTF16String(
+                                       IDS_CHAT_UI_SIDEBAR_TOOLTIP),
```

Comment:
We don't need to introduce a new string. We can modify the existing one to say 'Brave Leo'

Looks like in the current master we don't use `IDS_CHAT_UI_TITLE` anywhere but here

Reply:
done